modules/exploits/unix/webapp/joomla_comfields_sqli_rce.rb

```diff
@@ -0,0 +1,272 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
```

Comment:
I would add an include for https://github.com/rapid7/metasploit-framework/blob/master/lib/msf/core/exploit/http/joomla.rb

Reply:
Ready thanks